protocols.csv

```diff
@@ -1,15 +1,27 @@
-code,	size,	name
+code,	size,	name,	comment
 4,	32,	ip4
 6,	16,	tcp
 17,	16,	udp
 33,	16,	dccp
 41,	128,	ip6
+53,	V,	dns,	reserved for near-future use
+54,	V,	dns4,	reserved for near-future use
+55,	V,	dns6,	reserved for near-future use
 132,	16,	sctp
 301,	0,	udt
 302,	0,	utp
-421,	V,	ipfs
+400,	V,	unix
+420,	V,	p2p,	preferred over /ipfs
+421,	V,	ipfs,	equal to /p2p
+444,	96,	onion
+450,	V,	relay,	reserved for near-future use
 480,	0,	http
 443,	0,	https
-444,	96,	onion
-477, 0, ws
-275, 0, libp2p-webrtc-star
\ No newline at end of file
+477,	0,	ws
+478,	0,	wss
+275,	0,	libp2p-webrtc-star
+
+16383,	V,	exp-dns,	experimental /dns
```

Comment:
This is based on the highest value of a 2-byte uvarint: 2^14 - 1 = 11 1111 1111 1110 = 16383